### DISCUSSION TITLE
Windows compile instructions anyone ?

### DISCUSSION THREAD

**Discussion by CarloCattano** [monocasual/giada#506] [Tutorials and demos]
Im using Giada a lot recently on my raspberry pi . Tried cross-compiling but its really hard atm to find time to get it working . I thought on using windows to see the code and compile faster the changes , but Im having quite a bit of trouble with vs2019 to compile giada . When its not complaining about FLT window headers , its about vcpkg not doing what I ask him to do . Tried with CMake GUI and then termina, but still complains about headers (that are there) but Im too unexperienced to keep going .

Would be cool to dev on windows to make changes , currently testing a lot of UI changes for the raspberry 7inc screen :-)

Any help or hints would be appreciated

**Comment by gvnnz** [comment]
Hey @CarloCattano , compiling Giada on Windows should be easy, you can find the instructions [here](https://www.giadamusic.com/documentation-compiling-from-source). 

Actually we are compiling it on Windows every day in our GitHub Actions scripts, as you can see [here](https://github.com/monocasual/giada/blob/master/.github/workflows/ci.yml#L31). The only tricky part is installing dependencies: we do it with vcpkg in [a separate script](https://github.com/monocasual/giada/blob/master/.github/scripts/windows/install-deps.sh). Hope it helps.
